### PR TITLE
Implemented logging options

### PR DESCRIPTION
**Pull request by jjr840430**

### CODE REVIEW

File: common.c [hunk 1]
```diff
@@ -1809,3 +1826,159 @@ void *rtr_get_fuzzing_value(enum RTR_FUZZ_TYPE fuzz_type, void *param)
...
+				else
+					opt_val &= !bit_val;
+			}
+		}
+
+		if (opt_type == RTR_LOG_OPT_GRP)
+			g_logging_config.group_bitwise = opt_val;
+		else
+			g_logging_config.stracing_group_bitwise = opt_val;
+	} else if (opt_type == RTR_LOG_OPT_LEVEL) {
```
Comment (ronaldtse) [inline common.c]: @jjr840430 could we short circuit this here like this:

```c
	if (opt_type == RTR_LOG_OPT_GRP)
		g_logging_config.group_bitwise = opt_val;
	else
		g_logging_config.stracing_group_bitwise = opt_val;

	return;
}

if (opt_type == RTR_LOG_OPT_LEVEL) {
```

File: common.c [hunk 2]
```diff
@@ -1809,3 +1826,159 @@ void *rtr_get_fuzzing_value(enum RTR_FUZZ_TYPE fuzz_type, void *param)
...
+	if (stack_trace) {
+		/* check excluded list */
+		if (g_logging_config.stracing_disabled_funcs &&
+			rtr_check_config_token(event_info->function_name, g_logging_config.stracing_disabled_funcs, sep, &reverse) &&
+			!reverse)
+			return 0;
+
+		if ((event_info->function_group & g_logging_config.stracing_group_bitwise) == 0x00)
+			return 0;
+	} else {
```
Comment (ronaldtse) [inline common.c]: @jjr840430 could we short circuit this here like this:

```c
if (stack_trace) {
  ...
  return 1;
}

/* check fuzzing or redirection is applied */
if ((event_info->logging_level & RTR_LOG_LEVEL_FUZZ) || ...)
```

Comment (jjr840430) [inline common.c]: @ronaldtse just fixed. :)

**Comment by ronaldtse** [comment]
Thank you @jjr840430 !

**Pull request merged**